activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/test/java/org/activiti/cloud/services/query/rest/ProcessInstanceEntityTasksAdminControllerIT.java

```diff
@@ -66,9 +65,6 @@
     @MockBean
     private TaskRepository taskRepository;
 
-    @MockBean
-    private UserGroupManager userGroupManager;
-
     @MockBean
     private SecurityManager securityManager;
 
```

Comment:
Not sure why Codacy reported these Mockbeans as not needed, but if I remove them I get errors (failed to load application context) when running the test on local